torch/_export/serde/serialize.py

```diff
@@ -2311,6 +2358,8 @@ def rank_input(inp) -> Tuple[int, Optional[str], int]:
             return 3, spec.tensor_constant.tensor_constant_name, idx
         elif spec.type == "custom_obj":
             return 4, spec.custom_obj.custom_obj_name, idx
+        elif spec.type == "token":
+            return 0, None, idx
```

Comment:
@angelayi is token always ahead of other nodes for inputs and outputs?

Reply:
yeah